modules/oci-build/01_mod.mk

```diff
@@ -64,6 +64,10 @@ $(oci_build_targets): oci-build-%: ko-config-% | $(NEEDS_KO) $(NEEDS_GO) $(NEEDS
 		--push=false \
 		--bare
 
+	$(IMAGE_TOOL) append-layers \
```

Comment:
Does this break any sboms that we produce/ would produce in the future? I think it changes the digest, right?